kernel/src/client/default/filesystem.rs

```diff
@@ -107,11 +107,19 @@ impl<E: TaskExecutor> FileSystemClient for ObjectStoreFileSystemClient<E> {
                     };
                     let store = store.clone();
                     async move {
-                        if let Some(rng) = range {
-                            store.get_range(&path, rng).await
-                        } else {
-                            let result = store.get(&path).await?;
-                            result.bytes().await
+                        match url.scheme() {
+                            "http" | "https" => {
+                                // have to annotate type here or rustc can't figure it out
+                                Ok::<bytes::Bytes, Error>(reqwest::get(url).await?.bytes().await?)
```

Comment:
~~nit: not sure if we want to do here/now but prob would want to recycle a [Client] instead of doing `get` every time.~~

[Client]: https://docs.rs/reqwest/latest/reqwest/struct.Client.html

Reply:
oh wait i was thinking of the case below which you already used client for :) - disregard